src/actions/checkout-actions.js

```diff
@@ -103,7 +103,8 @@ export function authorizePayPalAndCreateOrder (payPalOrderID, paymentMethod) {
     // authorize the PayPal Order
     return dispatch(authorizePayPalOrder(payPalOrderID)).then(() => {
       const order = getState().order
-      if (!order.paymentResponseErrors.error.data) {
+      const paymentError = (order.paymentResponseErrors.error && order.paymentResponseErrors.error.data)
+      if (!paymentError) {
```

Comment:
What happens if we **do** get an error?

Reply:
its passed to the Payment summary component and displayed as a flash message -

![PayPal authorisation error message](https://user-images.githubusercontent.com/4486874/56891876-cc21dd00-6a75-11e9-9640-303e9fca50bb.png)